GDJS/Runtime/events-tools/inputtools.ts

```diff
@@ -68,18 +68,18 @@ namespace gdjs {
         RAlt: 2018,
         LSystem: 1091,
         RSystem: 2091,
-        /*"Menu": sf::Keyboard::Menu ,
```

Comment:
I went back to 2013 to understand why those values had been commented, but looks like they were added commented from the beginning :)
Something I'm missing or can we add them without risk?

Reply:
Can be added without risk.

Reply:
Can be added without risk.

Reply:
Can be added without risk!

Reply:
Can be added without risk!